spec/unit/packagers/msi_spec.rb

```diff
@@ -316,6 +316,23 @@ module Omnibus
       end
     end
 
+    describe '#fast_msi' do
```

Comment:
Seems like there may be alot of tests in this spec that could be influenced by fast_msi. Would it make sense to create shared examples and run through both implementations?